docs/src/pages/getting-started/migration/migration.react.mdx

```diff
@@ -0,0 +1,96 @@
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+import { Alert } from '@aws-amplify/ui-react';
+
+## Installation
+
+To begin, uninstall the previous `1.x` version of our library.
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm uninstall @aws-amplify/ui-components
+```
+
+</TabItem>
+<TabItem title="yarn">
+```shell
+yarn remove @aws-amplify/ui-components
+```
+
+</TabItem>
+</Tabs>
+
+After uninstalling our legacy authenticator, install the latest version of the authenticator.
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm install aws-amplify @aws-amplify/ui-react
+```
+
+</TabItem>
+<TabItem title="yarn">
+```shell
+yarn add aws-amplify @aws-amplify/ui-react
+```
+
+</TabItem>
+</Tabs>
+
+### Update and Usage
+
+Update the **App.js** with the new `Authenticator` and remove the old `AmplifyAuthenticator` as seen below:
+
+**App.js**
+
+```diff
+- import { AmplifyAuthenticator, AmplifySignOut } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+
++ import { Authenticator } from '@aws-amplify/ui-react';
++ import '@aws-amplify/ui-react/styles.css';
+
+import awsExports from './aws-exports';
+Amplify.configure(awsExports);
+
+ const App = () => (
+-  <AmplifyAuthenticator>
+-    <div>
+-      My App
+-      <AmplifySignOut />
+-    </div>
+-  </AmplifyAuthenticator>
++  <Authenticator>
++      {({ signOut, user }) => (
++        <main>
++          <h1>Hello {user.username}</h1>
++          <button onClick={signOut}>Sign out</button>
++        </main>
++      )}
++    </Authenticator>
+);
+
+```
+
+## Breaking changes
```

Comment:
This is great, thanks for including this section!
![](https://media.giphy.com/media/35pTQFiy14OeA/giphy.gif)